src/app/shared/filestack/preview/preview.component.spec.ts

```diff
@@ -51,19 +51,18 @@ describe('PreviewComponent', () => {
     spyOn(modalSpy, 'dismiss');
 
     component.file = { url: TEST_URL };
-    fixture.detectChanges();
-
-    const iconBtns: DebugElement[] = fixture.debugElement.queryAll(By.css('ion-icon'));
+    component.url = TEST_URL;
+    // fixture.detectChanges();
 
-    fixture.detectChanges();
-    const download: HTMLElement = iconBtns[0].nativeElement;
-    const close: HTMLElement = iconBtns[1].nativeElement;
+    // const iconBtns: DebugElement[] = fixture.debugElement.queryAll(By.css('ion-icon'));
```

Comment:
@shawnm0705 next time, to skip a test, maybe can just consider add "x" to before the "it". So it become "xit", the test would automatically get skipped.